protocol-definitions/DynamicConfig.yaml

```diff
@@ -1405,10 +1405,10 @@ methods:
             The WAN batch publisher configurations.
     response: {}
   - id: 19
-    name: addNamespaceConfig
+    name: addUserCodeNamespaceConfig
     since: 2.7
     doc: |
-      Adds a namespace configuration.
+      Adds a user code namespace configuration.
```

Comment:
I think the doc on the parameters needs to change as well

Reply:
The doc is already updated, as shown in the diff above.